### PR TITLE
feat(data-exploration): funnel results from insight data

### PR DESCRIPTION
**Pull request by thmsobrmlr**
## Problem

Displaying the funnel results is currently broken, see issue here https://github.com/PostHog/posthog/pull/14359.

## Changes

This PR renames the `results` from `dataNodeLogic` to `insightData` and uses it as source for `results` in `funnelDataLogic`. This fixes the issues seen in the first PR of all these stacked PRs.

I'm currently evaluating a singleton store for query / response [here](https://github.com/PostHog/posthog/pull/14507), with which the source of results in `funnelDataLogic` would then be rewritten.

## How did you test this code?

Clicking around.

### CODE REVIEW

File: frontend/src/scenes/funnels/funnelDataLogic.test.ts [hunk 1]
```diff
@@ -164,14 +164,9 @@ describe('funnelDataLogic', () => {
...
-    /**
-     * We set insightLogic.insight via a call to setInsight for data exploration,
-     * in future we should use the response of dataNodeLogic via a connected
-     * insightDataLogic.
-     */
-    describe('based on insightLogic', () => {
+    describe('based on insightDataLogic', () => {
         describe('results', () => {
-            it('with non-funnel insight', async () => {
+            it.skip('with non-funnel insight', async () => {
```
Comment (pauldambra) [inline frontend/src/scenes/funnels/funnelDataLogic.test.ts:169]: skip 🙈

File: frontend/src/scenes/insights/insightDataLogic.ts [hunk 2]
```diff
@@ -244,17 +244,17 @@ export const insightDataLogic = kea<insightDataLogicType>([
...
 
             actions.setInsight(
                 {
                     ...values.insight,
-                    result: response?.result,
-                    next: response?.next,
-                    // filters: queryNodeToFilter(query.source),
+                    result: insightData?.result,
+                    next: insightData?.next,
+                    filters: queryNodeToFilter(values.querySource),
```
Comment (pauldambra) [inline frontend/src/scenes/insights/insightDataLogic.ts:257]: is `queryNodeToFilter` safe for non-insight-viz queries? 

We have the case where you have a `PersonNode` for example, and want `filters` to be `{}` and `query` unchanged

**Review by pauldambra — COMMENTED** [review]
The test file seems unhappy 🙈 

It doesn't show the graph until refresh after save



other than that looks good...

So wonderful to have more and more tests on this!

File: frontend/src/scenes/funnels/funnelDataLogic.test.ts (hunk 1 above)
Comment (thmsobrmlr) [reply frontend/src/scenes/funnels/funnelDataLogic.test.ts:169]: The `.skip` is actually intentional here, as I've removed the condition that test if we currently have a funnel result (versus the result of another insight) and plan to re-add that with the data manager.

**Pull request merged**